src/internal.c

```diff
@@ -10537,7 +10537,7 @@ static int ProcessPeerCertParse(WOLFSSL* ssl, ProcPeerCertArgs* args,
     }
 
     /* Parse Certificate */
-    ret = ParseCertRelative(args->dCert, certType, verify, ssl->ctx->cm);
+    ret = ParseCertRelative(args->dCert, certType, verify, ssl->ctx->cm, 0);
```

Comment:
Why can't we leave the arg signature of `ParseCertRelative()` as it was, and make it a wrapper around a new `ParseCertRelativeLockOptional()` or somesuch?  I see a couple dozen of calls to `ParseCertRelative()` where the `lock` value is constant 0, and only 1 or 2 where it's not.

Similar thinking for `GetCA()` -- add a `GetCALockOptional()` for the few codepoints where the lock arg isn't constant 0?  Even for internal functions, it's worth keeping changes minimally intrusive, when the old form is already basically clean and clear.

Reply:
Sure, sounds good to me.